### ISSUE TITLE
Help text doesn't show up as Tooltip when hovering over stat on Character sheet

### ISSUE THREAD

**Issue by hawkerm** [fariapp/fari-app#363]
## 🚩 Describe the bug

Understand that help shows up in full character sheet on not on character page to save space, but would expect to see it as a tooltip or something on hover/tap of the text to be able to get assistance.

## 🔢 Steps to reproduce

1. Add help text to something on the character sheet
2. Go to Characters page
3. Hover mouse over that stat/block/bubble/etc...
4. Nothing happens...

## 🟢 Expected behavior

Expect to get a tooltip which contains the help text as a player aid.

## 🔴 Actual behavior

Nothing happens.

## 💅 Screenshots

In Character Page:

![image](https://user-images.githubusercontent.com/8959496/151732587-4a1649db-a0f6-46e3-a823-d260a36a5927.png)

In Character Sheet:

![image](https://user-images.githubusercontent.com/8959496/151732617-41507d6c-543b-4f8c-a72f-2f4d3a1cc4d4.png)

## 🖥 Desktop or Smartphone

- OS: Windows
- Browser: Edge
- Device: PC

**Comment by hawkerm** [comment]
Not familiar with react or structure of project, but based on a quick poke... I imagine if the `hideHelp` is true, then a tooltip attribute could be added to the box here with the help text (as it's a general property vs. specific to the type of block):

https://github.com/fariapp/fari-app/blob/81fbe0e7363f7461bd1b6042bbe6159e9549b6cd/lib/routes/Character/components/CharacterDialog/components/BlockByType.tsx#L104

**Comment by RPDeshaies** [comment]
What do you mean by "character page" ? Do you mean the character cards ?

**Comment by hawkerm** [comment]
> What do you mean by "character page" ? Do you mean the character cards ?

Yes, the character cards on the tab/page for characters in the main interface.
